packages/worker/src/worker.ts

```diff
@@ -700,7 +700,20 @@ export class Worker<T extends WorkerSpec = DefaultWorkerSpec> {
                   };
                   break;
                 }
-                const args = arrayFromPayloads(this.options.dataConverter, task.start?.input);
+                let args: any[];
```

Comment:
nit: let's use `unknown[]`, I'm still trying to remember to use unknown instead of any and TS has had that for a while now.